src/models/tree/build_tree.cu

```diff
@@ -259,37 +274,40 @@ __global__ static void __launch_bounds__(THREADS_PER_BLOCK)
   auto num_nodes = batch.NodesInBatch();
   int i          = rank / num_nodes;
   int j          = rank % num_nodes;
+  int output     = blockIdx.y;
```

Comment:
This function does not use the `n_ouputs` parameter. Looking at the launch specification for `scan_kernel,` I'm not sure how y represents the output, assuming this means vector leaf output target.
```
 const size_t blocks_y       = (num_features + features_per_warp - 1) / features_per_warp;
```
The `reduce_base_sums` launcher looks correct though.